sdk/communication/azure-communication-administration/azure/communication/administration/_shared/utils.py

```diff
@@ -68,3 +68,21 @@ def create_access_token(token):
         return AccessToken(token, datetime.fromtimestamp(payload['exp']).replace(tzinfo=TZ_UTC))
     except ValueError:
         raise ValueError(token_parse_err_msg)
+
+def get_authentication_policy(
+        endpoint, # type: str
+        credential # type: TokenCredential or str
+    ):
```

Comment:
it would also be useful to add a documentation for this method, and let the caller decide without checking the content of the method. Something like [this](https://github.com/Azure/azure-sdk-for-python/blob/0c5b1408d876464e268258d13cab43b965a58bc9/sdk/communication/azure-communication-administration/azure/communication/administration/_phone_number_administration_client.py#L115)